Makefile

```diff
@@ -26,11 +26,13 @@ fixup_locale:
 snag_docs: Documentation/man1/git-filter-repo.1 Documentation/html/git-filter-repo.html
 
 Documentation/man1/git-filter-repo.1:
-	mkdir -p man1
+	mkdir -p Documentation/man1
+	git branch --track docs remotes/origin/docs || true # ignore error in case the docs branch already exists
 	git show docs:man1/git-filter-repo.1 >Documentation/man1/git-filter-repo.1
```

Comment:
I would rather replace "docs" with "origin/docs" on the git show line than set up a local branch.

Reply:
Ahah that does sound much better :) ! My git-fu isn't up to snuff, I actually didn't even know about `git show` before.